examples/README.md

```diff
@@ -1694,6 +1694,29 @@ the controller and login nodes. Also since this blueprint doesn't use external
 IPs for compute nodes, one must needs to [set up cloud nat][cloudnat] and
 [set up iap][iap].
 
+Now, one need to update the blueprint to include shared vpc details. One should
+replace the host-project to the project in which shared vpc is created. In the
```

Comment:
Overall I think it could be more clear what user needs to update. 

1. s/host-project/`host_project_id`/
2. Add inline comment after yaml on L1703, L1704, L1716, L1717

Ex:
```
      subnetwork_name: <shared-subnetwork>  # update /w shared sub-net name
```